modules/ROOT/pages/revisionhistory.adoc

```diff
@@ -0,0 +1,116 @@
+
+== Interactive example
+
+liveDemo::{plugincode}[]
+
+include::partial$misc/admon-iframe-only.adoc[]
+
+== How it works
+
+The {pluginname} view is accessible via either the `revisionhistory` toolbar button or menu button within the `View` menu.
+
+The key components are:
+
+1. In the toolbar header, there are two buttons.
+* `Restore this version`: Set the selected version's content to the editor and close the view.
+* `Close`: Exit the view.
+
+2. The readonly diff view presents the changes between the selected version and its immediate predecessor, clearly highlighting for easy recognition. The changes are also color-coded for clarity:
+* Red: Removed content
+* Green: New content
+* Yellow: Content being modified. Modifications to HTML content implies attribute or format changes (bold, italic, etc)
+3. The revisions sidebar displays all available document versions. When a new version is selected, the diff view is updated accordingly.
+
+[NOTE]
+The default highlighting colors can be customized using the xref:revisionhistory_css_url[`+revisionhistory_css_url+`] option.
+
+[NOTE]
+The {pluginname} plugin interprets commented HTML as valid content, yet it disregards this content during the version comparison process. Consequently, a version containing solely commented content appears as empty in the user interface displaying the differential content.
+
+== Basic setup
+
+To setup the {pluginname} plugin user-interface in the editor:
+
+* add `{plugincode}` to the `plugins` option in the editor configuration;
+* add `{plugincode}` to the `toolbar` option in the editor configuration;
+* add `{plugincode}_fetch` option to the editor configuration;
+
+For example:
+
+[source,js]
+----
+tinymce.init({
+  selector: 'textarea',  // change this value according to your HTML
+  plugins: 'revisionhistory',
+  toolbar: 'revisionhistory',
+  // Required
+  revisionhistory_fetch: () => new Promise((resolve) => {
+    const revisions = [];  // Replace this with an API request to get saved versions
+    resolve(revisions);
+  })
+});
+----
+
+== Understanding initial and current versions
+
+When dealing with a new document, that has no saved versions, {pluginname} plugin will always typically maintain two versions: the `initial` and `current` versions.
+
+* Initial Version: this version is generated during the {productname} `Loaded` event, with the editor's content initialized.
+* Current Version: is created on opening the {pluginname} with the editor's current content. If included, it will become the `latest version` and placed at the top of the revisions list.
+
+[TIP]
+When the editor is initialized with content, both versions' content are the same. In that case, the {pluginname} only keeps the `current version`.
+
+For documents with saved versions, the `initial version` is disregarded by the {pluginname} assuming it's already part of the version history. The current version is discarded if its content matches the closest saved version.
```

Comment:
```suggestion
For documents with saved versions, the `current version` is discarded if its content matches the closest saved version.
```

Reply:
Removing `the initial version is disregarded...` would cause confusion when the initial version is not shown and there are saved versions.